test/inductor/test_max_autotune.py

```diff
@@ -930,6 +930,9 @@ def f(x, weight):
             act = opt_f(x, weight)
             self.assertTrue(torch.allclose(ref, act, atol=4 * 1e-3, rtol=4 * 1e-3))
 
+    @skipIfXpu(
```

Comment:
Shouldn't we have it xfail not skip then?

Reply:
See the issue.